gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/crud_service/flow/FlowCrudServiceImpl.java

```diff
@@ -55,6 +56,74 @@ public List<Flow> saveApiFlows(String apiId, List<Flow> flows) {
         return save(FlowReferenceType.API, apiId, flows);
     }
 
+    @Override
+    public List<Flow> getApiV4Flows(String apiId) {
```

Comment:
should we have only one method like following instead of two method that differ only by the reference type ? 🤔 

```java
    public List<Flow> getOrderedV4Flows(FlowReferenceType.API type, String id) {
         try {
            log.debug("Get flows for API {}", apiId);
            return flowRepository
                .findByReference(type, id)
                .stream()
                .sorted(Comparator.comparing(io.gravitee.repository.management.model.flow.Flow::getOrder))
                .map(FlowAdapter.INSTANCE::toFlowV4)
                .collect(Collectors.toList());
        } catch (TechnicalException ex) {
            final String error = "An error occurs while trying to get flows for API: " + apiId;
            log.error(error, ex);
            throw new TechnicalDomainException(error, ex);
    }
```

Reply:
I've kept the naming style of the existing functions `savePlanFlows` & `saveApiFlows`. Their names didn't shock me.  I think it's more appropriate to have the same rule. Either you always pass the type as a parameter, or you add it to the method name. I don't know if there are any conventions on this. 

But yes i can mutualise this part with a private `getV2(FlowReferenceType flowReferenceType, String referenceId)` same for V4